---
geometry_msgs/PoseStamped waypoint
int64 status